frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -181,7 +181,7 @@ const onChatScroll = function () {
     if (offsetTop + height <= curScrollBottom) {
       const bottomMessageCreatedHeight = msg.height
       const latestMessageCreatedHeight = this.currentChatRoomReadUntil?.createdHeight
-      if (!latestMessageCreatedHeight || latestMessageCreatedHeight <= bottomMessageCreatedHeight) {
+      if ((!latestMessageCreatedHeight || latestMessageCreatedHeight <= bottomMessageCreatedHeight) && !msg.pending) {
```

Comment:
This is not needed. msg.pending is already checked above in the line 177.
```
if (msg.pending || msg.hasFailed) continue
```

Reply:
This sounds right, it's already checked. I added the check to all places that were updating the last unread count.